galleries/users_explain/quick_start.py

```diff
@@ -558,8 +558,29 @@ def my_plotter(ax, data1, data2, param_dict):
 # control the size.  Finally, the colorbar will have default locators
 # and formatters appropriate to the norm.  These can be changed as for
 # other Axis objects.
+
+# %%
+# Patches
+# =======
```

Comment:
This seems good, but is this really the best place for it within this document?  Nothing nearby seems related?   Maybe put near annotations?

Reply:
This was arbitrary, I am definitely happy to move if there are more appropriate places. I wanted to give it a menu entry as this was mentioned in the original issue (#9967)

Reply:
I would actually go back a step and have the quick-start guide have a section on plotting methods in general that could include patches. Probably with a link to Plot Types to make it short. 

I think more detailed commentary belongs in https://matplotlib.org/devdocs/users/explain/artists/index.html.

Reply:
Do you suggest a new section at that page, or adding a new "Concept: Patches" page? I think I can definitely see the latter - sounds like a nice addition. I'm wondering if we want to have both - a quick mention at the quickstart and a more elaborate document under the page you mentioned? I have no strong feelings either way

Reply:
If it was just me, I'd add to quick-start a very short section on plot-types, and include patches as a special case of lower level plot types that need `add_artist`.  I'd add a section in user/explain/artists on "Patches". We may want to rename user/explain/artists to make it more clear this refers to plot elements, but not sure what that name would be.